client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -48,13 +50,23 @@ class TaxonomyManagerListItem extends Component {
 		this.props.onClick();
 	};
 
-	deleteItem= () => {
+	deleteItem = () => {
 		this.setState( {
 			popoverMenuOpen: false
 		} );
 		this.props.onDelete();
 	};
 
+	getTaxonomyLink() {
+		const { taxonomy, siteUrl, slug } = this.props;
+		let taxonomyBase = taxonomy;
+
+		if ( taxonomy === 'post_tag' ) {
+			taxonomyBase = 'tag';
+		}
```

Comment:
We are starting to have these kind of specific checks inside of our components. We have this one, and we have the `default_category` setting for Categories. 

I wonder if it's not the right time to create a `taxonomy-utils` with small functions like: `getTaxonomyUrlSlug` `getTaxonomyDefaultAttributeName`

(maybe later)